### DISCUSSION TITLE
App: Export as Human Readable / Displayed

### DISCUSSION THREAD

**Discussion by PacifistRiot** [directus/directus#10448] [Feature Requests]
Currently we have the ability implemented to export raw data from the app as CSV, JSON, and XML but I would love to have the ability to be able to export the data as human readable or displayed information. That is, phrasing the data to what the display settings are set to for the field or relationship.

For example, if I have a collection for Printers that include a many to one field for Toner with the display settings telling it to display that field as the Model of the Toner, instead of exporting the collection with the raw data of "12", export the collection with the display data of "HP Toner XYZ".

Hope that makes sense!

**Comment by benhaynes** [comment]
Absolutely! What we need to do is add a secondary output to the displays. So in addition to the current Vue/HTML output, there would be an additional plaintext version that could be used for exports another purposes throughout the app (client-side column sorting based on the display output?). 👍

**Reply by mslonina** [reply]
Would it be possible to have hookable exports here? So that one can provide different export type via extensions?

**Reply by rijkvanzanten** [reply]
@mslonina Yeah, that'd be a good idea! We also have to think through how to pass options to these export extensions 🤔 

Right now, we're doing `?export=csv`. Maybe we can add a `?export=csv&export_options[delimiter]="|"`

**Comment by azrikahar** [comment]
#12201 should make this possible now once it's released!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
